cmd/protoc-gen-go-grpc/protoc-gen-go-grpc_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex # Debugging enabled.
+
+die() {
+    echo "$@" >&2
+    exit 1
+}
+
+trap "git clean --force --quiet" EXIT
+
+WD="$(dirname $0)"
+
+protoc \
+    --go_out=. \
+    --go_opt=paths=source_relative \
+    --go-grpc_out=. \
+    --go-grpc_opt=paths=source_relative \
```

Comment:
This will run the version of protoc-gen-go-grpc that's in your path?  I think you need to do:

```sh
go build .
PATH="$(pwd):${PATH}" protoc --go-grpc_out=. ......
```

OR find a way to run `protoc` where you point it at the binary directly instead of having it run out of your path using magic to parse flags.

Reply:
Done and tested: https://github.com/arvindbr8/grpc-go/actions/runs/8546553547/job/23417150415?pr=5